poems.md

```diff
@@ -0,0 +1,344 @@
+#Maya Angelou#
+<br>
+<br>
+**Awaking in New York** <br>
+*Shaker, Why Don’t You Sing, 1983* <br>
+<br>
+Curtains forcing their will <br>
+against the wind, <br>
+children sleep, <br>
+exchanging dreams with <br>  
+seraphim. The city <br>
+drags itself awake on <br>
+subway straps; and <br>
+I, an alarm, awake as a <br>
+rumor of war, <br>
+lie stretching into dawn, <br>   
+unasked and unheeded. <br>
+
+**The Mothering Blackness** <br>
+*Just Give Me a Cool Drink of Water Fore I Die, 1971* <br>
+<br>
+She came home running <br>
+       back to the mothering blackness <br> 
+       deep in the smothering blackness<br>
+white tears icicle gold plains of her face <br>  
+       She came home running <br>
+<br>
+
+She came down creeping <br>
+       here to the black arms waiting <br>
+       now to the warm heart waiting <br>
+rime of alien dreams befrosts her rich brown face <br>   
+       She came down creeping <br>
+<br>
+She came home blameless <br>
+       black yet as Hagar’s daughter <br>
+       tall as was Sheba’s daughter <br>
+threats of northern winds die on the desert’s face<br>   
+       She came home blameless</p> <br>
```

Comment:
There's an orphan end of paragraph tag here (/p) that's causing a long gap between these two poems in the .md view https://github.com/suwebdev/orange-dream/pull/14/files?short_path=e310c07#diff-e310c073b6b3c903994c5931182d6324
I'm not sure this is anything we need to fix at this point since we're mostly just looking for the content. I'm voting for a merge either way. Thanks for gathering these poems Mary!